torch/_functorch/_aot_autograd/schemas.py

```diff
@@ -267,6 +267,11 @@ class ViewAndMutationMeta:
     # raised
     deterministic: Optional[bool] = None
 
+    # Map of effect type (ex. _EffectType.ORDERED) to token.  If there are
+    # side-effectful operators, FunctionalTensorMode will populate this
+    # dictionary telling us how many tokens we will need during tracing.
+    tokens: Dict[Any, torch.Tensor] = field(default_factory=dict)
```

Comment:
are the values here guaranteed to be FakeTensors? (and if so maybe we should make the type annotation `[Any, FakeTensor]`, to make it clear we aren't secretly holding on to tensor memory)

Reply:
Hmm, I guess in the context of aot_autograd they're always FakeTensors?

Reply:
Oh but sometimes theyre FunctionalTensors?

Reply:
hmmm yeah fair, i guess we can't easily make the annotation more specific